app/webpacker/components/RegistrationsV2/Register/RegistrationRequirements.jsx

```diff
@@ -0,0 +1,102 @@
+import { UiIcon } from '@thewca/wca-components';
+import { marked } from 'marked';
+import React, {
+  useContext, useEffect, useMemo, useState,
+} from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Accordion,
+  Button,
+  Form,
+  Message,
+  Segment,
+  Transition,
+} from 'semantic-ui-react';
+import { CompetitionContext } from '../Context/competition_context';
+
+export default function RegistrationRequirements({ nextStep }) {
+  const { competitionInfo } = useContext(CompetitionContext);
+
+  const [generalInfoAcknowledged, setGeneralInfoAcknowledged] = useState(false);
+  const [regRequirementsAcknowledged, setRegRequirementsAcknowledged] = useState(false);
+
+  const [showRegRequirements, setShowRegRequirements] = useState(false);
+
+  const { t } = useTranslation();
+
+  const setFromCheckbox = (data, setState) => {
+    const { checked } = data;
+    setState(checked);
+  };
```

Comment:
I think we have hooks for that in the monolith React code, under `webpacker/lib/hooks`